server/galaxyls/tests/files/complex_inputs_01_command.xml

```diff
@@ -0,0 +1,38 @@
+<command><![CDATA[
+
+## Auto-generated command section
+## TODO: please review and edit this section as needed
+
+## Inputs
+
+#if str( \$c1.c1_action ) == "a1":
```

Comment:
this  `\` is wrong it means that the `$` is not intepreted by cheetag but by bash

Reply:
Sorry, I forgot to mention that this is a TextMate snippet, so, it has to escape the `$` in many places because it is also part of the snippet syntax. But the end result after it is inserted in the document should be `#if str( $c1.c1_action ) == "a1"`